integration-core/src/main/java/org/symphonyoss/integration/core/bridge/IntegrationBridgeImpl.java

```diff
@@ -67,24 +69,36 @@
 
   @Override
   public List<Message> sendMessage(IntegrationInstance instance, String integrationUser,
-      List<String> streams, String message) {
+      List<String> streams, String message) throws RemoteApiException {
     List<Message> result = new ArrayList<>();
 
+    boolean wasMessageSent = Boolean.FALSE;
```

Comment:
You might remove this boolean flag and use the size of the result list to check if one message was delivered successfully.